langlib/lang.value/src/main/ballerina/value.bal

```diff
@@ -80,6 +90,11 @@ public isolated function cloneReadOnly(CloneableType  v) returns CloneableType &
 # - if a record type descriptor specifies default values, these will be used
 #   to supply any missing members
 #
+# ```ballerina
+# anydata userIds = [12, 19, 56, 75, 103, 119, 121];
+# int[] copyOfUserIds = check userIds.cloneWithType();
+# ```
+#
```

Comment:
We need another example with having parameter `t`.

Reply:
Yes, this is mentioned in the doc too. 

`userIds.cloneWithType(int[])` won't work since we can't use `int[]` as an expression.

E.g.,
```ballerina
var employee = check userIds.cloneWithType(Employee);
```

Reply:
Applicable for some other functions too.